var/spack/repos/builtin/packages/dsqss/package.py

```diff
@@ -45,3 +45,23 @@ def cmake_args(self):
         ]
 
         return args
+
+    def test(self):
+        test01 = find(self.prefix.share, '01_spindimer')[0]
+        copy(join_path(test01, 'std.toml'), '.')
+        # prepare
+        pythonexe = self.spec['python'].command.path
+        opts = [self.spec.prefix.bin.dla_pre, 'std.toml']
+        self.run_test(pythonexe, options=opts)
+        # (mpi) run
+        opts = []
+        if self.spec.satisfies('+mpi'):
+            exe_name = self.spec['mpi'].prefix.bin.mpirun
+            opts.extend(['-n', '1'])
+            opts.append(join_path(self.prefix.bin, 'dla'))
+        else:
+            exe_name = 'dla'
+        opts.append('param.in')
+        expected = ['R ene = -3.74300000e-01 2.96344394e-03']
```

Comment:
Is this the only text output to `sample.log`?

Reply:
No. please see installed reference (share/dsqss/2.0.3/samples/dla/01_spindimer/ref/sample.log).
However, sample.log does not include message like "Finished" or "All done.".
So I checked only return code and `R ene` line. 
Only this line was checked in the tutorial.

Reply:
That's fine.  I was just wondering.